### PR TITLE
Fix unnecessary livestream api calls in channel page

### PR DESCRIPTION
**Pull request by btzr-io**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I added a line describing my change to CHANGELOG.md
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #6632 
Changelog update: #6735

### CODE REVIEW

File: ui/component/livestreamLink/view.jsx [hunk 1]
```diff
@@ -18,9 +18,11 @@ export default function LivestreamLink(props: Props) {
   const [livestreamClaim, setLivestreamClaim] = React.useState(false);
   const [isLivestreaming, setIsLivestreaming] = React.useState(false);
   const livestreamChannelId = (channelClaim && channelClaim.claim_id) || ''; // TODO: fail in a safer way, probably
+  const isChannelEmpty = !channelClaim || !channelClaim.meta || !channelClaim.meta.claims_in_channel;
 
   React.useEffect(() => {
-    if (livestreamChannelId) {
+    // Don't search empty channels
+    if (livestreamChannelId && !isChannelEmpty) {
       Lbry.claim_search({
```
Comment (jessopb) [inline ui/component/livestreamLink/view.jsx:26]: Nice.
This might be faster with page_size=1 and no_totals?
@tzarebczan

Comment (btzr-io) [reply ui/component/livestreamLink/view.jsx:26]: Is there any difference between `claim_list` and `claim_search` ?
https://lbry.tech/api/sdk#claim_list

Comment (jessopb) [reply ui/component/livestreamLink/view.jsx:26]: claim list is specifically one's own publishes.

Comment (jessopb) [reply ui/component/livestreamLink/view.jsx:26]: page=1 is a given - page_size=1 would limit results. Not sure how big of a deal it would be.

Comment (btzr-io) [reply ui/component/livestreamLink/view.jsx:26]: Can you publish something while livestreaming ?
I think will be safer to check more than one claim.

Comment (tzarebczan) [reply ui/component/livestreamLink/view.jsx:26]: the "has no source param" ensures it's only the live streams that are returned. And right now we use the latest one you have published.

**Review by jessopb — APPROVED** [review]
:+1:

**Comment by btzr-io** [comment]
Changelog update: #6735

**Comment by btzr-io** [comment]
Any reason for not merging this yet ?

**Pull request merged**